apps/web/pages/apps/installed.tsx

```diff
@@ -141,14 +141,6 @@ function ConnectOrDisconnectIntegrationButton(props: {
       </div>
     );
   }
-  /** We don't need to "Connect", just show that it's installed */
-  if (props.isGlobal) {
-    return (
-      <div className="truncate px-3 py-2">
-        <h3 className="text-sm font-medium text-gray-700">{t("installed")}</h3>
-      </div>
-    );
-  }
```

Comment:
We want to get rid of this since global apps are no longer defaulted to installed

Reply:
If we plan to remove this we should make sure that all global apps are installable. Like Cal Video and the others

Reply:
So for this I decided to make Jitsi and Huddle not global. My definition of a global app would be something that just comes with cal.com kind of like Cal Video. I reverted my other changes where we remove checks for global apps.